### DISCUSSION TITLE
posts/2024-08-01-ai-t-cells/

### DISCUSSION THREAD

**Discussion by giscus** [racheltho/rachel-blog#16] [Announcements]
# posts/2024-08-01-ai-t-cells/

Different neural nets can be used to predict T cell binding, and it is unclear which is best.

https://rachel.fast.ai/posts/2024-08-01-ai-t-cells/

**Comment by tobsecret** [comment]
I really really like this writeup! Especially liked the off-target binding example that illustrated that this is one of those problems where the threshold for what can be considered useful is extremely high. 

I think this def warrants a 4th part about the exact challenges of determining what counts as binding and how much binding we actually want.

There's also a question about what an effective strategy for collecting the needed data should be.

One other interesting aspect I find is that while the space of possible peptides is very large it is restricted by the total allowed length of what fits into the TCR binding pocket.

I've recently looked at some published methods (MixTCRpred, PanPep, TSpred) and have concluded that for our projects these are not useful due to their abysmal performance on unseen peptides. The expected rate of FPs is just so high that it's not even worth running.
